tests/bento_service_examples/keras_classifier.py

```diff
@@ -4,20 +4,28 @@
 from bentoml.adapters import JsonInput
 from bentoml.frameworks.keras import KerasModelArtifact
 
+import tensorflow as tf
+
+tf.compat.v1.disable_eager_execution()
+
 
 @bentoml.env(infer_pip_packages=True)
 @bentoml.artifacts(
     [
         KerasModelArtifact('model'),
         # TODO: #1698 set store_as_json_and_weights to True after the issue is fixed
-        KerasModelArtifact('model2', store_as_json_and_weights=False),
+        KerasModelArtifact('model2', store_as_json_and_weights=True),
     ]
 )
 class KerasClassifier(bentoml.BentoService):
     @bentoml.api(input=JsonInput(), batch=True)
     def predict(self, jsons):
-        return self.artifacts.model.predict(np.array(jsons))
+        raw_artifact = self._artifacts['model']
+        with raw_artifact.graph.as_default(), raw_artifact.sess.as_default():
+            return self.artifacts.model.predict(np.array(jsons))
```

Comment:
This is ugly. We want to access the session/graph associated with the keras model, so we need to access the raw `KerasModelArtifact` instead of `self.artifacts.model`.

Maybe we can create a customized context manager to make this look better?